_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
     }
 }
 ```
+<h3 id="tracker_updated">
+    tracker:updated
+</h3>
+
+```json
+{
+    "topic" : "tracker:updated",
+    "message" : {
+        "order_ref" : "9099",
+        "order_unique_id" : "100000018",
+        "shipment_unique_id" : "100000007",
+        "warehouse_id": 1,
+        "package_id" : 99,
+        "source" : "shopify",
+        "carrier" : "fedex",
+        "service" : "FEDEX_2_DAY",
+        "number" : "794657040105",
```

Comment:
Please use "tracking_number" to be consistent with other places and avoid confusion in the future with other numbers.